client/my-sites/email/email-different-domain-owner-message/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTranslate } from 'i18n-calypso';
+import TrackComponentView from 'calypso/lib/analytics/track-component-view';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+
+type EmailDifferentDomainOwnerMessageProps = {
+	source: string;
+};
+
+export const EmailDifferentDomainOwnerMessage = (
+	props: EmailDifferentDomainOwnerMessageProps
+) => {
+	const { source } = props;
+
+	const translate = useTranslate();
+
+	const onClickLink = ( eventType: 'login' | 'support' ) => {
+		const properties = {
+			action: eventType,
+			source,
+		};
+
+		recordTracksEvent( `calypso_email_providers_nonowner_click`, properties );
+	};
+
+	const translateOptions = {
+		components: {
+			contactSupportLink: (
+				<a
+					href="https://wordpress.com/help/contact"
+					onClick={ () => onClickLink( 'support' ) }
+					rel="noopener noreferrer"
+					target="_blank"
+				/>
+			),
+		},
+	};
+
+	return (
+		<>
+			<TrackComponentView
+				eventName="calypso_email_providers_nonowner_impression"
```

Comment:
Same remark about this event name, which should be refined to match the name of this component.

Reply:
This is meant to be the same event name. It's easier for us to create funnels and simply filter by property to see the users' response to each warning message.